### PR TITLE
feature/95/write test for PageWrapper component

### LINKED ISSUE #95: (SP: 1) Write unit test for "PageWrapper" component
### Component unit test
Unit test for "**Page wrapper**" component
Scenaries descriptions:
- [ ] Children of the component should be rendered
[Link to component](https://github.com/Space2Study-UA-1125/frontEnd/blob/develop/src/components/page-wrapper/PageWrapper.jsx)

Current coverage:
Image

### PR DESCRIPTION
**Pull request by Wiret02**
Wrote a test for PageWrapper component
image

### CODE REVIEW

**Review by iva-stasia — APPROVED** [review]

**Review by holus-bolus — APPROVED** [review]

**Review by StasRusnak1 — APPROVED** [review]

**Review by VictoriaRodik — APPROVED** [review]

File: tests/unit/components/page-wrapper/PageWrapper.spec.jsx
Comment (vlad-khrychov) [inline tests/unit/components/page-wrapper/PageWrapper.spec.jsx:1]: Add `describe` as in all other tests.

File: tests/unit/components/page-wrapper/PageWrapper.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import PageWrapper from '~/components/page-wrapper/PageWrapper'
+
+afterEach(() => {
+  cleanup()
+})
+
+it('renders children correctly', () => {
+  const childText = 'Test Child Component'
+
+  const { getByText } = render(
```
Comment (vlad-khrychov) [inline tests/unit/components/page-wrapper/PageWrapper.spec.jsx]: Maybe call render in `beforeEach`?

File: tests/unit/components/page-wrapper/PageWrapper.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    </PageWrapper>
+  )
+
+  expect(getByText(childText)).toBeInTheDocument()
+})
+
+it('closes modal on unmount', () => {
+  const { unmount } = render(<PageWrapper>Test Content</PageWrapper>)
+
+  unmount()
```
Comment (vlad-khrychov) [inline tests/unit/components/page-wrapper/PageWrapper.spec.jsx]: Add expect. We can check that `childText` will not be in the document.

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

**Review by ashfox21 — APPROVED** [review]

File: tests/unit/components/page-wrapper/PageWrapper.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { render, cleanup, screen } from '@testing-library/react'
+import { beforeEach, describe } from 'vitest'
```
Comment (vlad-khrychov) [inline tests/unit/components/page-wrapper/PageWrapper.spec.jsx]:
```suggestion
```

**Review by vlad-khrychov — APPROVED** [review]
You can merge after deleting of 1 row.

**Pull request merged**